dali/operators/reader/loader/indexed_file_loader.h

```diff
@@ -171,8 +169,8 @@ class IndexedFileLoader : public Loader<CPUBackend, Tensor<CPUBackend>, true> {
       } else {
         tensor.Resize({size}, DALI_UINT8);
 
-        int64 n_read = current_file_->Read(reinterpret_cast<uint8_t*>(tensor.raw_mutable_data()),
-                            size);
+        int64 n_read =
+            current_file_->Read(reinterpret_cast<uint8_t*>(tensor.raw_mutable_data()), size);
```

Comment:
```suggestion
            current_file_->Read(static_cast<uint8_t*>(tensor.raw_mutable_data()), size);
```
...while you're at it.